examples/lib/intermediateHtmlStepUi.js

```diff
@@ -60,11 +60,29 @@ function IntermediateHtmlStepUi(_sequencer, step, options) {
     var parser = new DOMParser();
     var addStepUI = stepUI();
     addStepUI = parser.parseFromString(addStepUI, "text/html").querySelector("div")
-    step.ui
+
+    var toggleDiv = () => {
+      $(step.ui.querySelector('.insertDiv')).fadeToggle(200).collapse('toggle')
+      if ($(step.ui.querySelector('.insert-text')).css('display') != "none"){
+        $(step.ui.querySelector('.insert-text')).fadeToggle(200, () => {$(step.ui.querySelector('.no-insert-text')).fadeToggle(200)})
+      }
+      else {
+        $(step.ui.querySelector('.no-insert-text')).fadeToggle(200, () => {$(step.ui.querySelector('.insert-text')).fadeToggle(200)})
```

Comment:
I think we should migrate to normal functions here instead of arrow functions as it may not be supported by some browsers.

Reply:
I had asked @jywarren about this, he said I can use these if they are supported and they are.

I think all modern (>IE) browsers support arrow functions and template literals. Including EdgeSafari everything.

https://kangax.github.io/compat-table/es6/